arwen/elrondapi/elrondei.go

```diff
@@ -3136,19 +3146,31 @@ func v1_4_getOriginalTxHash(context unsafe.Pointer, dataOffset int32) {
 	runtime := arwen.GetRuntimeContext(context)
 	metering := arwen.GetMeteringContext(context)
 
-	gasToUse := metering.GasSchedule().ElrondAPICost.GetBlockHash
+	gasToUse := metering.GasSchedule().ElrondAPICost.GetOriginalTxHash
 	metering.UseGas(gasToUse)
 
 	err := runtime.MemStore(dataOffset, runtime.GetOriginalTxHash())
 	_ = arwen.WithFault(err, context, runtime.ElrondAPIErrorShouldFailExecution())
 }
 
+//export v1_4_getCurrentTxHash
```

Comment:
I think all the new references to v1.4 should be changed to v1.5 as this feature will be included in the v1.5 variant of the arwen.

Reply:
The `v1_4` prefix for API functions must be changed for all functions at once and it should be a completely separate operation. Changing it right now will complicate things. It will be changed after `master` is on `v1_5`.

Reply:
great